virtualizarr/manifests/array_api.py

```diff
@@ -239,6 +240,50 @@ def _check_same_shapes(shapes: List[Tuple[int, ...]]) -> None:
             )
 
 
+@implements(np.expand_dims)
+def expand_dims(x: "ManifestArray", /, *, axis: int = 0) -> "ManifestArray":
+    """Expands the shape of an array by inserting a new axis (dimension) of size one at the position specified by axis."""
+    # this is just a special case of stacking
+    return stack([x], axis=axis)
+
+
+@implements(np.broadcast_to)
+def broadcast_to(x: "ManifestArray", /, shape: Tuple[int, ...]) -> "ManifestArray":
+    """
+    Broadcasts an array to a specified shape, by either manipulating chunk keys or copying chunk manifest entries.
+    """
+    print(f"requested broadcasting array from shape {x.shape} to shape {shape}")
```

Comment:
Replace with log statement?

Reply:
Oops thanks for catching that! Removed in https://github.com/TomNicholas/VirtualiZarr/commit/c0ef78c93b1745218ce4286b25f9d83ee50f860b